### DISCUSSION TITLE
Fixed Table Header with Relative Table Height

### DISCUSSION THREAD

**Discussion by taltshaked** [ant-design-blazor/ant-design-blazor#2511] [Q&A] [answered]
Hello,
I want to use the ScrollY parameter in the Table component so that the header is fixed when scrolling vertically.
Doing so works, however, some users have different resolutions so specifying a hard-coded ScrollY value will result in different percentage of table space utilization.
Is there a way to get that fixed table header and still have the table utilizing the same percentage of vertical space used by the table on every resolution? (e.g., make the table fill up up to 90% of the page's vertical space regardless of the resolution or system font size)

Thanks in advance

**Comment by ElderJames** [comment, marked as answer]
Do you want dynamic height or fixed height?  If you're dealing only with resolution, you can use em/rem units.

**Reply by taltshaked** [reply]
I want the height to be relative to the height of the user's screen and window, or to keep the same margins from above and below the table, regardless of the window size or resolution/font-size, while keeping the table header fixed.
How do I do that?
Thank you

**Reply by ElderJames** [reply]
I think the only way to listen for window size and set the value of ScrollY is through JS interop.

**Reply by taltshaked** [reply]
Thank you. I hope to see this feature becoming resolution-independent in future releases, ideally as a property "`FixHeader`" rather using the "`ScrollY`".
